test/test_jiterator.py

```diff
@@ -0,0 +1,168 @@
+# Owner(s): ["module: cuda"]
+
+import torch
+from torch.cuda.jiterator import create_jit_fn
+import sys
+import unittest
+from torch.testing._internal.common_utils import TestCase, parametrize, run_tests, instantiate_parametrized_tests, TEST_WITH_ROCM
+
+TEST_CUDA = torch.cuda.is_available()
```

Comment:
This can be imported from common_utils:

https://github.com/pytorch/pytorch/blob/201ddafc22e22c387b4cd654f397e05354d73d09/torch/testing/_internal/common_utils.py#L763

Reply:
right.